tests/gnrc_sock_dns/README.md

```diff
@@ -0,0 +1,25 @@
+# Overview
+
+This folder contains a test application for RIOT's sock-based DNS client.
+
+# How to test with native
+
+Setup up a tap interface:
+
+    # sudo ip tuntap add dev riot0 mode tap user $(id -u -n)
```

Comment:
Seriously? Most Linux distributions (at least all I ever used) use this notation when ever you call `su` (see example from my current Ubuntu setup):

```
[mlenders@sarajevo RIOT]<3 sh
$ sudo su
[sudo] password for mlenders: 
root@sarajevo:/home/mlenders/Repositories/RIOT-OS/RIOT# 
```